CHANGELOG.md

```diff
@@ -1,5 +1,268 @@
 # Lotus changelog
 
+# v1.21.0-rc1 / 2023-03-21
+
+This is an optional but highly recommended feature release of Lotus. It includes numerous improvements and enhancements for node operators, ETH RPC-providers and storage providers.
+
+## ☢️ Upgrade Warnings ☢️
+
+Before upgrading to this feature release read carefully through these bullet points:
+
+- Starting from this release, the SplitStore feature is automatically activated on new nodes. However, for existing Lotus users, you need to explicitly configure SplitStore by uncommenting the `EnableSplitstore` option in your `config.toml` file. To enable SplitStore, set `EnableSplitstore=true`, and to disable it, set `EnableSplitstore=false`. **It's important to note that your Lotus node will not start unless this configuration is properly set. Set it to false if you are running a full archival node!**
+- This feature release requires a **minimum Go version of v1.19.7 or higher to successfully build Lotus**. Additionally, Go version v1.20 and higher is now also supported.
```

Comment:
Wait, didn't we end up pulling in libp2p upgrade in the end? master works on Go 1.20

Reply:
Correct, that is included in this release also. Maybe the wording here is a bit off? But personally I think it makes it relatively clear that go version of 1.20 is also supported.

> This feature release requires a minimum Go version of v1.19.7 or higher to successfully build Lotus Additionally, Go version v1.20 and higher is now also supported.